scene/resources/material.cpp

```diff
@@ -727,6 +727,35 @@ void BaseMaterial3D::_update_shader() {
 			break; // Internal value, skip.
 	}
 
+	switch (depth_function) {
+		case DEPTH_FUNCTION_LESS_OR_EQUAL:
+			code += ",depth_function_less_or_equal";
```

Comment:
Here is where the missing space comes from other material shader generators include the space after the comma